src/test/regress/input/multi_create_schema.source

```diff
@@ -8,7 +8,7 @@ CREATE TABLE nation (
 	n_name char(25) not null,
 	n_regionkey integer not null,
 	n_comment varchar(152));
-SELECT create_distributed_table('tpch.nation', 'n_nationkey', 'append');
+SELECT create_distributed_table('tpch.nation', 'n_nationkey', 'hash', shard_count := 1);
```

Comment:
well, very minor, but we could even make this reference table, as it is more common in the schemas?

Reply:
upon closer inspection this table is not used, and the whole test file seems redundant